ethcore/res/ethereum/constantinople_test.json

```diff
@@ -6,11 +6,12 @@
 				"minimumDifficulty": "0x020000",
 				"difficultyBoundDivisor": "0x0800",
 				"durationLimit": "0x0d",
-				"blockReward": "0x4563918244F40000",
+				"blockReward": "0x29A2241AF62C0000",
 				"homesteadTransition": "0x0",
-				"eip649Reward": "0x29A2241AF62C0000",
 				"eip100bTransition": "0x0",
-				"eip649Transition": "0x0"
+				"difficultyBombDelays": {
+					"0": 3000000
```

Comment:
can you add the 1234 portion here for constantinople?

Reply:
I think this constantinople_test is totally unsynch (see the name), currently it's basically the byzantium one. I could indeed be a good idea to get it ready for actual constantinople (if all pr are merged).